Tests/ResourcesTests/MyViewController.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  MyViewController.swift
+//  SwifterSwift
+//
+//  Created by Vato Kostava on 7/1/20.
+//  Copyright © 2020 SwifterSwift
+//
+
+import Foundation
```

Comment:
Fix this for Linux builds
```suggestion
#if canImport(UIKit) && !os(watchOS)
import UIKit
```